Mathlib/CategoryTheory/Sites/Whiskering.lean

```diff
@@ -157,4 +159,43 @@ lemma sheafCompose_id : sheafCompose_map (F := F) J (𝟙 _) = 𝟙 _ := rfl
 lemma sheafCompose_comp :
     sheafCompose_map J (η ≫ γ) = sheafCompose_map J η ≫ sheafCompose_map J γ := rfl
 
+end Multicospan
+
+section Preserves
+
+variable [PreservesLimitsOfSize.{v₁} F] [PreservesLimitsOfSize.{v₁} G]
+  [PreservesLimitsOfSize.{v₁} H] (J)
```

Comment:
It would be more convenient to the human reader to make explicit the second relevant universe parameter:
```suggestion
variable [PreservesLimitsOfSize.{v₁, max u₁ v₁} F] [PreservesLimitsOfSize.{v₁, max u₁ v₁} G]
  [PreservesLimitsOfSize.{v₁, max u₁ v₁} H] (J)
```